packages/fxa-profile-server/docs/API.md

```diff
@@ -391,3 +395,83 @@ Delete cached profile data for user `:uid`
 [ecosystem_anon_id-post]: #post-v1ecosystem_anon_id
 [oauth]: https://github.com/mozilla/fxa-oauth-server
 [cache_delete]: #delete-v1cache:uid
+
+### GET /v1/\_core_profile
```

Comment:
@bbangert  @chenba  Addded the important information related to the enhancement of API.md in the fxa-profile-server